extensions/sources/base/src/main/java/io/smallrye/ext/config/source/base/BaseConfigSource.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2019 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.smallrye.ext.config.source.base;
```

Comment:
I'd probably drop the `base` from the package name, as it's core functionality to config sources it doesn't need a sub package

Reply:
Done